packages/storybook-addon-designs/src/index.ts

```diff
@@ -23,6 +23,8 @@ export const withDesign = makeDecorator({
  */
 export const config = (c: Config | Config[]) => c
 
+export { PARAM_KEY, Config };
```

Comment:
You can just-rename-and-export a variable with the below code.

```ts
import { ParameterName } from './addon'

export { ParameterName as PARAM_KEY }
```

Changing `addon.ts#ParameterName` would break internal naming consistency (both PascalCase and UPPER_SNAKE_CASE for string constants). I prefer just adding a public interface. Also, with renaming at export time, you no longer need to include 55ed43999d2249288b93ed03f7c2077f28675f30 (#54 ) !